tools/pelela-cli/templates/base-template-for-cli/styles.css

```
@@ -1 +1,44 @@
-/* Put global styles here */
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap');
```

Stylelint (17.10.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap'" (import-notation)

(import-notation)